misaligned.cpp

```diff
@@ -1,21 +1,33 @@
 #include <iostream>
 #include <assert.h>
 
-int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
-    for(i = 0; i < 5; i++) {
-        for(j = 0; j < 5; j++) {
-            std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
-        }
-    }
-    return i * j;
+int computePairNumber(int major, int minor)
+{
+	return major * 5 + minor;
 }
 
-int main() {
-    int result = printColorMap();
-    assert(result == 25);
-    std::cout << "All is well (maybe!)\n";
-    return 0;
-}
\ No newline at end of file
+int printColorMap() 
+{
+	
+	const char* majorColor[] = { "White", "Red", "Black", "Yellow", "Violet" };
+	const char* minorColor[] = { "Blue", "Orange", "Green", "Brown", "Slate" };
+	int i = 0, j = 0;
+	for (i = 0; i < 5; i++) 
+	{
+		for (j = 0; j < 5; j++) 
+		{
+			std::cout << computePairNumber(i,j) << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
+		}
+	}
+	return i * j;
+}
+
+int main()
+{
+	int result = printColorMap();
+	assert(result == 25);
+	computePairNumber(0, 0);

```

Comment:
There's a call to `computePairNumber` inside the assert, so we don't need to call it again